mobile/tsconfig.json

```diff
@@ -43,7 +43,13 @@
     // "paths": {},                           /* A series of entries which re-map imports to lookup locations relative to the 'baseUrl'. */
```

Comment:
I spent a little bit of time trying to get `paths` to work instead of hardcoding dependencies to @burstjs in your `package.json`, but I failed miserably. That said, i think that's the _right_ way to do it, see the angular-wallet for an example. This way you can always pull in burstjs from the monorepo instead of npm. This should enable you to add new features to the lib and use them in your app without publishing to npm, and allows you to use new unpublished burstjs features before they are published to npm.

Reply:
See https://github.com/burst-apps-team/phoenix/blob/develop/web/angular-wallet/tsconfig.json#L14

Reply:
Actually, this solve a RN TS bug with React types, but ok, I agree

Reply:
Tried, but encountered haste map error again

Reply:
Yea same here :(